support/oidc-discovery-provider/registration_api.go

```diff
@@ -0,0 +1,147 @@
+package main
+
+import (
+	"context"
+	"crypto/x509"
+	"sync"
+	"time"
+
+	"github.com/andres-erbsen/clock"
+	"github.com/gogo/protobuf/proto"
+	"github.com/sirupsen/logrus"
+	"github.com/spiffe/spire/proto/spire/api/registration"
+	"github.com/spiffe/spire/proto/spire/common"
+	"github.com/zeebo/errs"
+	"google.golang.org/grpc"
+	"gopkg.in/square/go-jose.v2"
+)
+
+const (
+	DefaultRegistrationAPIPollInterval = time.Second * 5
```

Comment:
This is documented (and enforced in config) to default to 10 seconds
```suggestion
	DefaultRegistrationAPIPollInterval = time.Second * 10
```